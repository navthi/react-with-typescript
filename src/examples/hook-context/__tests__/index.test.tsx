import * as React from 'react';
import { shallow } from 'enzyme';

import Counter from '../index';

describe('examples/context', () => {
  it('renders count', () => {
    const wrapper = shallow(<Counter />);
    wrapper.setState({ count: 42 });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
