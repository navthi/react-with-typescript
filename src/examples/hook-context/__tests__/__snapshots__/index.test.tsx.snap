// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples/context renders count 1`] = `"<div><h1>Count: 42</h1><button>+1</button><button>-1</button></div>"`;
